### PR TITLE
rm IName sheet from item, comment, proposals resource types API CHANGE

### PR DESCRIPTION
**Pull request by joka**

### CODE REVIEW

File: src/adhocracy_core/adhocracy_core/resources/item.py [hunk 1]
```diff
@@ -33,8 +31,7 @@ def create_initial_content_for_item(context, registry, options):
 
 item_meta = pool_meta._replace(
     iresource=IItem,
-    basic_sheets=[adhocracy_core.sheets.name.IName,
```
Comment (pallix) [inline src/adhocracy_core/adhocracy_core/resources/item.py]: so is a migration required here?

**Comment by pallix** [comment]
6 functionals tests are failing on this branch

File: src/adhocracy_core/adhocracy_core/resources/item.py (hunk 1 above)
Comment (joka) [inline src/adhocracy_core/adhocracy_core/resources/item.py]: > so is a migration required here?

yup

**Comment by xi** [comment]
Why is the `IName` sheet removed when changing to autonaming? I would have expected that it would simply become read-only.

**Comment by joka** [comment]
> Why is the IName sheet removed when changing to autonaming? 

The frontend team wanted to have it  that way.

>I would have expected that it would simply become read-only.

**Comment by joka** [comment]
Test fixed

**Comment by pallix** [comment]
Migration is written, should be tested on staging first.

**Comment by nidico** [comment]
Looks fine on staging.

**Pull request merged**